src/Tasks/Microsoft.NET.Build.Tasks/msbuildExtensions-ver/Microsoft.Common.targets/ImportAfter/Microsoft.NET.Build.Extensions.targets

```diff
@@ -0,0 +1,7 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <MicrosoftNETBuildExtensionsTargets Condition="'$(MicrosoftNETBuildExtensionsTargets)' == ''">$(MSBuildExtensionsPath)\Microsoft.NET.Build.Extensions\Microsoft.NET.Build.Extensions.targets</MicrosoftNETBuildExtensionsTargets>
```

Comment:
We have approval, folks specifically pushed for the design change to put this stuff inbox.

I share your concern for performance, see below.

As far as risk assessment, what is your recommendation?  I discussed various options with @dsplaisted and this is where we landed.